docker/intel-dgpu-driver.Dockerfile

```diff
@@ -10,13 +10,23 @@ WORKDIR /build/
 # RUN git clone -b 23WW06.5_555_MAIN --single-branch https://github.com/intel-gpu/intel-gpu-cse-backports.git && cd intel-gpu-cse-backports && export OS_TYPE=rhel_8 && export OS_VERSION="8.6" && make modules && make modules_install
 
 # Building pmt(VSEC) driver
-RUN git clone -b 23WW06.5_555_MAIN --single-branch https://github.com/intel-gpu/intel-gpu-pmt-backports.git && cd intel-gpu-pmt-backports && export OS_TYPE=rhel_8 && export OS_VERSION="8.6" && make modules && make modules_install
+RUN git clone -b 23WW06.5_555_MAIN --single-branch https://github.com/intel-gpu/intel-gpu-pmt-backports.git \
+    && cd intel-gpu-pmt-backports \
+    && export OS_TYPE=rhel_8 && export OS_VERSION="8.6" \
+    && make modules && make modules_install
 
```

Comment:
Do you think it is good for us to keep each command a line? That is more readable.

Reply:
@uMartinXu To make the text more readable, we have broken the long line into multiple lines. Also tried to group similar commands/functionalities on the same line to avoid confusion between their specific purposes (E.g. all `make` on same line). Does it look good or want to change it to new lines?

Reply:
to put the relevant commands into one line is also readable to me.